samples/multiple-inheritance/Core_6/Producer/Contracts/MyEvent.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Publisher.Contracts
+{
+    using Subscriber1.Contracts;
+    using Subscriber2.Contracts;
+
+    #region publisher-contracts
+    class MyEvent :
+        Consumer1Contract,
+        Consumer2Contract
+    {
```

Comment:
I tried but couldn't come up with a real life use case, when we agree on https://github.com/Particular/docs.particular.net/pull/2290#issuecomment-266915497 we might find one